### PR TITLE
added packages/conf-haveged/conf-havaged.1.0.0

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

File: packages/conf-haveged/conf-havaged.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+homepage: "https://linux.die.net/man/8/haveged"
+author: "A. Seznec and N. Sendrier"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL-1.0-or-later"
+depexts: [
+# ["havaged"] {os = "openbsd"} # not in openbsd?!
+# ["havaged"] {os-family = "debian"} # not in Debian?!
```
Comment (kit-ty-kate) [inline packages/conf-haveged/conf-havaged.1.0.0/opam]:
```suggestion
  ["havaged"] {os-family = "debian"}
```
It seems to exist https://packages.debian.org/buster/haveged

File: packages/conf-haveged/conf-havaged.1.0.0/opam [hunk 2]
```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+homepage: "https://linux.die.net/man/8/haveged"
+author: "A. Seznec and N. Sendrier"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+license: "GPL-1.0-or-later"
+depexts: [
```
Comment (kit-ty-kate) [inline packages/conf-haveged/conf-havaged.1.0.0/opam]:
```suggestion
build: ["haveged" "--help"]
depexts: [
```
Maybe that could help detect if it is actually installed quicker

Comment (kit-ty-kate) [reply packages/conf-haveged/conf-havaged.1.0.0/opam]: Or maybe not, it seems installed in /usr/sbin on debian which might be inaccessible for non-privileged users

File: packages/conf-haveged/conf-havaged.1.0.0/opam (hunk 1 above)
Comment (UnixJunkie) [reply packages/conf-haveged/conf-havaged.1.0.0/opam]: ok, thanks

**Comment by kit-ty-kate** [comment]
ohh... Is it `haveged` or `havaged`? Both seem to be in the package (package name vs. homepage vs. depexts vs. debian package name). Could you clear that out?

**Comment by UnixJunkie** [comment]
Sorry, there was a typo in many places effectively.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**